dev/start

```diff
@@ -256,6 +257,15 @@ env_args="$env_args -e CONJUR_AUTHENTICATORS=$enabled_authenticators"
 
 docker-compose up -d --no-deps $services
 
+# If the enabled authenticators changed after initial Conjur configuration,
+# then docker-compose recreates the container to set the environment variable,
+# and we need to restart the Conjur server process.
```

Comment:
Nice catch! In which scenario did you see this? Does it really matter that we run the server again before loading the policy with alice? It seems that it's the only thing happening before killing the process again.

Reply:
Micah actually caught it and when you run `./start --authn-ldap` for example. If we don't run the service again the user needs to manually authenticate in order to add the policy with Alice. The difference between the two runs is highlighted [here](https://github.com/cyberark/conjur/pull/1582#issuecomment-638229844)

Reply:
Oh nice! I remember seeing this behaviour once. I think it happens only with `authn-ldap`. thanks!